crates/bevy_pbr/src/render/mesh.rs

```diff
@@ -2190,23 +2203,24 @@ impl<P: PhaseItem, const I: usize> RenderCommand<P> for SetMeshViewBindGroup<I>
             view_ssr,
             view_environment_map,
             mesh_view_bind_group,
+            maybe_oit_layers_count_offset,
         ): ROQueryItem<'w, Self::ViewQuery>,
         _entity: Option<()>,
         _: SystemParamItem<'w, '_, Self::Param>,
         pass: &mut TrackedRenderPass<'w>,
     ) -> RenderCommandResult {
-        pass.set_bind_group(
-            I,
-            &mesh_view_bind_group.value,
-            &[
-                view_uniform.offset,
-                view_lights.offset,
-                view_fog.offset,
-                **view_light_probes,
-                **view_ssr,
-                **view_environment_map,
-            ],
-        );
+        let mut offsets = vec![
```

Comment:
Can we use a smallvec/arrayvec here? Allocating in the hotpath is a bad idea.

Reply:
I switched to a smallvec and looks like things still work as expected.